recipes/pybind11/all/conanfile.py

```diff
@@ -16,17 +16,25 @@ class PyBind11Conan(ConanFile):
 
     _source_subfolder = "source_subfolder"
 
+    _cmake = None
+
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
 
+    def requirements(self):
+        # self.requires("cpython/x.y/z") or self.requires("pypy/x.y/z")
```

Comment:
```suggestion
        # TODO: self.requires("cpython/x.y/z") or self.requires("pypy/x.y/z")
```

Make easier to be found in the future